webapplication/sip/settings.py

```diff
@@ -165,6 +165,7 @@
 
 STATIC_URL = '/static/'
 STATIC_ROOT = os.path.join(BASE_DIR, 'static')
+MEDIA_PATH = 'static'
```

Comment:
This should be /media

Reply:
And /media should be mapped to the external folder in docker-compose-*.yml (like db volume)

Reply:
done
but if I use `/media` then ```
ImageField's 'upload_to' argument must be a relative path, not an absolute path.
        HINT: Remove the leading slash.
```
that's why `media`